docs/install.md

```diff
@@ -14,15 +14,54 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-# Install 
+# Installation
 
 There are three ways to use Triton Model Analyzer:
 
-1. Using the `pip`:
+1. Building the Dockerfile:
```

Comment:
I think we can encourage them to use the Triton SDK container instead of building Docker.